src/Microsoft.DocAsCode.App/RunServe.cs

```diff
@@ -43,11 +47,104 @@ public static void Exec(string folder, string host, int? port)
             Console.WriteLine($"Serving \"{folder}\" on {url}. Press Ctrl+C to shut down.");
             using var app = builder.Build();
             app.UseFileServer(fileServerOptions);
-            app.Run();
+
+            if (openBrowser)
+            {
+                string relativePath = openBrowserRelativePath;
+                var launchUrl = string.IsNullOrEmpty(relativePath)
+                    ? url
+                    : Path.Combine(url, ResolveOutputHtmlRelativePath(folder, relativePath));
+
+                // Start web server.
+                app.Start();
+
+                // Launch browser process.
+
+                Console.WriteLine($"Launching browser with url: {url}.");
+                using var process = LaunchBrowser(launchUrl);
+
+                // Wait until server exited.
+                app.WaitForShutdown();
+
+                // process object is disposed. (Note:Launched process remain running.after dispose)
```

Comment:
nit: if we aren't using the `process` object here and disposing it doesn't not terminate the launched process, consider make `LaunchBrowser` return `void`.

Reply:
Fixed in latest commit.

Originally, I've try to called `process.Kill()` here.
and want to ensure Dispose method called because `Process` might contains unmanaged resources.
Anyway. when serve command completed. docfx.exe is exit. There is no problems change this helper method to return void.